includes/rest-api/class-sensei-rest-api-setup-wizard-controller.php

```diff
@@ -349,9 +349,11 @@ public function submit_features( $data ) {
 
 		$this->mark_step_complete( 'features' );
 
+		$selected_features = $data['selected'];
```

Comment:
Also about the changes from the other PR, https://github.com/Automattic/sensei/pull/3190, could we remove this change from this PR?

Reply:
Let's merge #3190 first, and we'll clean this one up after.

Reply:
So these duplicates should be resolved now with the merge & 345c437.